nixos/modules/services/web-apps/mealie.nix

```diff
@@ -39,6 +39,15 @@ in
       };
     };
 
+    extraOptions = lib.mkOption {
```

Comment:
I don't like that it's called `extraOptions` where it doesn't set an option but really an argument to the commandline of `gunicorn` specifically.
Idk the standard way of calling these options, but something like `gunicornArguments` can still sound better to my ear

Reply:
Should already be the case :D